src/pages/sykmeldinger/SykmeldingerPage.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import { Normaltekst } from 'nav-frontend-typografi';
-import { AlertStripeAdvarsel } from 'nav-frontend-alertstriper';
+import {Normaltekst} from 'nav-frontend-typografi';
```

Comment:
Eg har formateringa til å vere `import { Normaltekst }` 😄  Er vel litt ymse her i sykmeldinger, men vi kan jo sette opp regla for slikt etterkvart?

Reply:
ah ja dette er grunnen til at vi trenger husky og lint-staged etc etc... :weary: